examples/atomic-counters/atomic-counters.go

```diff
@@ -7,43 +7,40 @@
 
 package main
 
-import "fmt"
-import "time"
-import "sync/atomic"
+import (
```

Comment:
We should probably have a convention across the site for when we import multiple packages with a single statement, vs one package on each line.

In the original version of the site I tended to used the latter to reduce the length of examples (in terms of lines).

@eliben do you have an opinion on this?

Reply:
Ah, looking through the examples I think it's my fault that a number of them switched to line-per-import to this style :-/

My editor is doing this automatically for whenever there's more than a single import. In fact, this is how the official [goimports](https://godoc.org/golang.org/x/tools/cmd/goimports) tool is doing it, which is why most editor plugins do it these days.

Thinking about what's common in the Go world, I think that the `import ( ... )` style is more common for when there's more than a single import. Examples:

1. Tour of Go: https://tour.golang.org/welcome/4
2. Effective Go: https://golang.org/doc/effective_go.html
3. Go standard library https://github.com/golang/go/blob/master/src/encoding/json/encode.go#L13
4. Go compiler https://github.com/golang/go/blob/master/src/cmd/compile/internal/ssa/block.go#L8
5. k8s: https://github.com/kubernetes/kubernetes/blob/master/pkg/auth/nodeidentifier/default.go#L19
6. Docker: https://github.com/docker/engine/blob/master/cli/required.go

etc.

But I see the point you're making about line per import being slightly (one line) shorter.

I'm somewhat in favor of using the more prevalent style, esp. given that many editors/IDEs pretty much force it these days, but am fine with whatever decision you want to make here. I'll be happy to send a PR to fix all samples to a single style, once we decide.

Reply:
I think consistently using `import ( ... )`  when there's more than important makes sense given the Go convention. A separate PR for that sounds good!

Reply:
OK, opened #272 to track this